docs/cert-manager.md

```diff
@@ -0,0 +1,52 @@
+# MinIO tenant with cert-manager [![Slack](https://slack.min.io/slack?type=svg)](https://slack.min.io)
+
+This document explains how to deploy a MinIO tenant using certificates generated by [cert-manager](https://cert-manager.io/).
+
+## Getting Started
+
+### Prerequisites
+
+- Kubernetes version `+v1.18`, for a complete list of supported versions by cert-manager see https://cert-manager.io/docs/installation/supported-releases/
```

Comment:
```suggestion
- Kubernetes version `+v1.19`. While cert-manager supports [earlier K8s versions](https://cert-manager.io/docs/installation/supported-releases/), the MinIO Operator requires 1.19 or later.
```

Reply:
We still support the unsupported `1.18` since some customers still have that version of k8s @ravindk89

Reply:
I think docs can be prescriptive - if we happen to support 1.18 that's fine, but we're designing for non-EOL. Otherwise users have an expectation of things continuing to work on 1.18 and we can grow the overall userbase on that deploy.

Will defer to you on this though. We should update readme.md if we want to officially call out 1.18 as the minver.